### PR TITLE
typo in feature

### PR DESCRIPTION
**Pull request by jamesgorrie**
Oops

### CODE REVIEW

**Comment by paperboyo** [comment]
This feature is a bug!

File: media-api/app/controllers/MediaApi.scala [hunk 1]
```diff
@@ -321,7 +321,7 @@ object MediaApi extends Controller with ArgoHelpers {
       "obsbizcash", "obscomment", "obsfestivals", "obsfocus", "obsforeignnews", "obsgalleries",
       "obshomenews", "obsmastheads", "obssports", "obssuppliments",
```
Comment (theefer) [inline media-api/app/controllers/MediaApi.scala]: Is it really "obssuppliments"?

**Comment by jamesgorrie** [comment]
@paperboyo Why?
@theefer it's taken directly from Picdar.

**Comment by paperboyo** [comment]
> it's taken directly from Picdar

But it's not really connected to Picdar any more, right? I, for one, like when people fix my grammar...

Unfortunately, they cannot fix my sense of humour, which is the answer to another of @jamesgorrie's questions.

**Comment by jamesgorrie** [comment]
Ah wait, @theefer - that was [spotted and fixed](https://github.com/guardian/grid/blob/master/media-api/app/controllers/MediaApi.scala#L322). My bad.

**Comment by theefer** [comment]
Cool

**Pull request merged**